### DISCUSSION TITLE
Server return Internal Error

### DISCUSSION THREAD

**Discussion by noinhotnn** [seaweedfs/seaweedfs#3885] [Q&A] [answered]
`Seems volume 3925 content was missing somehow. ` as @chrislusf  said.
Is there any way to recover data and what are the possible causes of this condition?
**Filer**
![image](https://user-images.githubusercontent.com/13940206/197093673-60b07526-c8a0-450f-a59a-2b9e7f4d0519.png)
**Get Object**
![image](https://user-images.githubusercontent.com/13940206/197093620-f239ca06-5bfa-4781-b3c0-dded0d56a12c.png)
**Browser**
![image](https://user-images.githubusercontent.com/13940206/197094296-9b9d0b4e-af47-4cbb-a260-c6e1927cee09.png)
**Volume same in two rack**
![image](https://user-images.githubusercontent.com/13940206/197096147-4da8de9b-4e25-47e0-af42-6bb101052ba1.png)

**Comment by chrislusf** [comment]
Do you find the `3925.dat` file?

**Reply by noinhotnn** [reply]
File exists
![image](https://user-images.githubusercontent.com/13940206/197428275-6f238bb3-073e-420e-a369-194d19519c1f.png)
Other Volume
`E1021 09:33:41.796522 common.go:288 processRangeRequest: read chunk: http://10.0.10.183:8085/3925,0b3e3e6c7b6c7aee?readDeleted=true: 404 Not Found
I1021 09:34:44.435483 filechunk_manifest.go:197 read http://10.0.10.184:8085/3906,0af6d8062fe6ebcd failed, err: http://10.0.10.184:8085/3906,0af6d8062fe6ebcd?readDeleted=true: 404 Not Found
I1021 09:34:44.435483 filechunk_manifest.go:197 read http://10.0.10.183:8085/3906,0af6d8062fe6ebcd failed, err: http://10.0.10.183:8085/3906,0af6d8062fe6ebcd?readDeleted=true: 404 Not Found
I1021 09:34:44.436480 filechunk_manifest.go:203 retry reading in 1s
I1021 09:34:45.436506 filechunk_manifest.go:197 read http://10.0.10.184:8085/3906,0af6d8062fe6ebcd failed, err: http://10.0.10.184:8085/3906,0af6d8062fe6ebcd?readDeleted=true: 404 Not Found
I1021 09:34:45.436506 filechunk_manifest.go:197 read http://10.0.10.183:8085/3906,0af6d8062fe6ebcd failed, err: http://10.0.10.183:8085/3906,0af6d8062fe6ebcd?readDeleted=true: 404 Not Found
I1021 09:34:45.437506 filechunk_manifest.go:203 retry reading in 1.5s
I1021 09:34:46.938548 filechunk_manifest.go:197 read http://10.0.10.184:8085/3906,0af6d8062fe6ebcd failed, err: http://10.0.10.184:8085/3906,0af6d8062fe6ebcd?readDeleted=true: 404 Not Found
I1021 09:34:46.938548 filechunk_manifest.go:197 read http://10.0.10.183:8085/3906,0af6d8062fe6ebcd failed, err: http://10.0.10.183:8085/3906,0af6d8062fe6ebcd?readDeleted=true: 404 Not Found
I1021 09:34:46.939548 filechunk_manifest.go:203 retry reading in 2.25s
I1021 09:34:49.191296 filechunk_manifest.go:197 read http://10.0.10.184:8085/3906,0af6d8062fe6ebcd failed, err: http://10.0.10.184:8085/3906,0af6d8062fe6ebcd?readDeleted=true: 404 Not Found
I1021 09:34:49.192295 filechunk_manifest.go:197 read http://10.0.10.183:8085/3906,0af6d8062fe6ebcd failed, err: http://10.0.10.183:8085/3906,0af6d8062fe6ebcd?readDeleted=true: 404 Not Found
I1021 09:34:49.193294 filechunk_manifest.go:203 retry reading in 3.375s
I1021 09:34:52.569382 filechunk_manifest.go:197 read http://10.0.10.184:8085/3906,0af6d8062fe6ebcd failed, err: http://10.0.10.184:8085/3906,0af6d8062fe6ebcd?readDeleted=true: 404 Not Found
I1021 09:34:52.569382 filechunk_manifest.go:197 read http://10.0.10.183:8085/3906,0af6d8062fe6ebcd failed, err: http://10.0.10.183:8085/3906,0af6d8062fe6ebcd?readDeleted=true: 404 Not Found
I1021 09:34:52.570383 filechunk_manifest.go:203 retry reading in 5.0625s
E1021 09:34:57.634294 filer_server_handlers_read.go:249 failed to stream content /buckets/2022/09/15/ThueXe/74175/15H03961_C.txt: read chunk: http://10.0.10.183:8085/3906,0af6d8062fe6ebcd?readDeleted=true: 404 Not Found
E1021 09:34:57.634294 common.go:288 processRangeRequest: read chunk: http://10.0.10.183:8085/3906,0af6d8062fe6ebcd?readDeleted=true: 404 Not Found
2022/10/21 09:34:57 http: superfluous response.WriteHeader call from github.com/seaweedfs/seaweedfs/weed/server.processRangeRequest (common.go:289)
I1021 09:35:25.752170 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:25.752170 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:25.753169 filechunk_manifest.go:203 retry reading in 1s
I1021 09:35:26.753378 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:26.753378 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:26.754375 filechunk_manifest.go:203 retry reading in 1.5s
I1021 09:35:28.255673 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:28.255673 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:28.256673 filechunk_manifest.go:203 retry reading in 2.25s
I1021 09:35:30.507067 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:30.507067 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:30.508064 filechunk_manifest.go:203 retry reading in 3.375s
I1021 09:35:33.884211 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:33.884211 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:33.885211 filechunk_manifest.go:203 retry reading in 5.0625s
E1021 09:35:38.949414 filer_server_handlers_read.go:249 failed to stream content /buckets/2022/09/13/ThueXe/74175/15H04947_C.txt: read chunk: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
E1021 09:35:38.949414 common.go:288 processRangeRequest: read chunk: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:39.363021 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:39.364021 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:39.365024 filechunk_manifest.go:203 retry reading in 1s
I1021 09:35:40.366661 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:40.366661 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:40.367653 filechunk_manifest.go:203 retry reading in 1.5s
I1021 09:35:41.867993 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:41.867993 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:41.869991 filechunk_manifest.go:203 retry reading in 2.25s
I1021 09:35:44.120569 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:44.120569 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:44.121567 filechunk_manifest.go:203 retry reading in 3.375s
I1021 09:35:47.496476 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:47.496476 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:47.497477 filechunk_manifest.go:203 retry reading in 5.0625s
E1021 09:35:52.560856 filer_server_handlers_read.go:249 failed to stream content /buckets/2022/09/13/ThueXe/74175/15H04947_C.txt: read chunk: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
E1021 09:35:52.560856 common.go:288 processRangeRequest: read chunk: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:54.175411 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:54.175411 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:54.176412 filechunk_manifest.go:203 retry reading in 1s
I1021 09:35:55.178312 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:55.178312 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:55.179316 filechunk_manifest.go:203 retry reading in 1.5s
I1021 09:35:56.681202 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:56.681202 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:56.682202 filechunk_manifest.go:203 retry reading in 2.25s
I1021 09:35:58.932339 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:58.933335 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:35:58.933335 filechunk_manifest.go:203 retry reading in 3.375s
I1021 09:36:02.309229 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:02.309229 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:02.310227 filechunk_manifest.go:203 retry reading in 5.0625s
E1021 09:36:07.372913 filer_server_handlers_read.go:249 failed to stream content /buckets/2022/09/13/ThueXe/74175/15H04947_C.txt: read chunk: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
E1021 09:36:07.372913 common.go:288 processRangeRequest: read chunk: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:13.784344 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:13.785344 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:13.786344 filechunk_manifest.go:203 retry reading in 1s
I1021 09:36:14.787175 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:14.788172 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:14.788172 filechunk_manifest.go:203 retry reading in 1.5s
I1021 09:36:16.288562 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:16.288562 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:16.289562 filechunk_manifest.go:203 retry reading in 2.25s
I1021 09:36:18.539763 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:18.541763 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:18.547770 filechunk_manifest.go:203 retry reading in 3.375s
I1021 09:36:21.923132 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:21.924126 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:21.924126 filechunk_manifest.go:203 retry reading in 5.0625s
E1021 09:36:26.987647 filer_server_handlers_read.go:249 failed to stream content /buckets/2022/09/13/ThueXe/74175/15H04947_C.txt: read chunk: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
E1021 09:36:26.987647 common.go:288 processRangeRequest: read chunk: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:52.596846 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:52.597842 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:52.598844 filechunk_manifest.go:203 retry reading in 1s
I1021 09:36:53.599632 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:53.599632 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:53.600632 filechunk_manifest.go:203 retry reading in 1.5s
I1021 09:36:55.102313 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:55.103316 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:55.103316 filechunk_manifest.go:203 retry reading in 2.25s
I1021 09:36:57.354806 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:57.354806 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:36:57.355822 filechunk_manifest.go:203 retry reading in 3.375s
I1021 09:37:00.731868 filechunk_manifest.go:197 read http://10.0.10.183:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.183:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:37:00.731868 filechunk_manifest.go:197 read http://10.0.10.184:8085/3679,0a5b679ad399c2f3 failed, err: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
I1021 09:37:00.732865 filechunk_manifest.go:203 retry reading in 5.0625s
E1021 09:37:05.795648 filer_server_handlers_read.go:249 failed to stream content /buckets/2022/09/13/ThueXe/74175/15H04947_C.txt: read chunk: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found
E1021 09:37:05.795648 common.go:288 processRangeRequest: read chunk: http://10.0.10.184:8085/3679,0a5b679ad399c2f3?readDeleted=true: 404 Not Found`

**Comment by chrislusf** [comment, marked as answer]
better avoid running volume servers on windows.

**Reply by noinhotnn** [reply]
Thank you. I will try on linux and will report back later.
